src/content/releases/all_releases.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "All Releases"
++++
+
+## Supported Releases
+The following OVN release series are still receiving upstream support.
+
+### OVN 23.06
+For more details, see [23.06 ](../23.06)
+
+
+
+### OVN 23.03
+For more details, see [23.03 ](../23.03)
+
+
+
+### OVN 22.12
+For more details, see [22.12 ](../22.12)
+
+
+
+### OVN 22.09
+For more details, see [22.09 ](../22.09)
+
+
+
+### OVN 22.03
+For more details, see [22.03 ](../22.03)
+
+
+## Unsupported Releases
```

Comment:
Nit: Is "No Longer Supported Releases" more accurate than "Unsupported Releases"?

Reply:
Thanks for the feedback Dumitru. That's a bad link to break since it's the first page people see. I will get that fixed.

And with regards to "Unsupported Releases" vs "No Longer Supported Releases", I can make that change as well.